model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -1342,7 +1342,11 @@ message StandardArtifacts {
   enum Roles {
     // A URN for staging-to role.
     // payload: ArtifactStagingToRolePayload
-    STAGING_TO  = 0 [(beam_urn) = "beam:artifact:role:staging_to:v1"];
+    STAGING_TO        = 0 [(beam_urn) = "beam:artifact:role:staging_to:v1"];
+
+    // A URN for pypi-install-requirement role.
+    // payload: None
+    PYPI_INSTALL_REQUIREMENT  = 1 [(beam_urn) = "beam:artifact:role:pypi_install_requirement:v1"];
```

Comment:
It's unclear here whether this is for requirements.txt files or actual tarballs that should be installed.

Could we call this either PYTHON_PACKAGES or PIP_REQUIREMENTS_FILE?

Reply:
Use PIP_REQUIREMENTS_FILE instead of PYPI_INSTALL_REQUIREMENT